js/schedule.js

```diff
@@ -847,10 +906,33 @@ function Schedule(CUSTOM_CONFIG) {
     window.scrollTo(0, 0);
   }
 
+  //Manages state of timeblocks and stores that in localStorage
   schedule.toggleTimeblock = function(timeblockContainer) {
-    timeblockContainer.find(".timeblock-header")
+    timeblockContainer.toggleClass('expanded')
+                      .find(".timeblock-header")
                       .find('.fa').toggleClass('fa-chevron-circle-left fa-chevron-circle-down');
-    schedule.animateBlockToggle(timeblockContainer.find(".sessions-container"));
+    timeblockContainer.find('.sessions-container').slideToggle();
+
+    var timeblockIsOpen = timeblockContainer.hasClass('expanded');
+    var openTimeblocks = JSON.parse(localStorage.getItem('timeblock-states')) || [] ;
```

Comment:
Not sure I need to, since `getItem` will return null if the key doesn't exist, and the whole expression has the `[]` fallback..